js/docx/smoke/api_uni_color/to_json.js

```diff
@@ -0,0 +1,29 @@
+builder.CreateFile("docx");
+let oDocument = Api.GetDocument();
+let oParagraph = oDocument.GetElement(0);
+let oRGBColor = Api.CreateRGBColor(255, 111, 61);
```

Comment:
We are testing ApiUniColor because the final JSON expects a value of 'uniColor' in the respective field regardless of the original color type that was used when creating the document. Therefore, even if we use RGBColor directly, it does not impact the expected behavior we are testing in this case. This is because RGBColor is a subclass of ApiUniColor and when it is used, the final color type in the JSON remains 'uniColor'. This particular test verifies this functionality.

Reply:
I think we need to get confirmation from the developer that it is not possible to create an object of UniColor class